modular_skyrat/modules/ashwalkers/code/effects/ash_ritual.dm

```diff
@@ -0,0 +1,211 @@
+/datum/ash_ritual
+	/// the name of the ritual
+	var/name = "Summon Coders"
+
+	/// the components necessary for a successful ritual
+	var/atom/north_ritual_component
```

Comment:
i am going to continue to hound you about the assoc list that'd look something like this
`list("North" = /obj/component/one, "East" = /obj/component/two, "South" = null, "West" = null)`
where null are spots that don't require anything

Reply:
get the free gbp when you refactor... or I can later on.